resources/js/components/HelloChild.jsx

```diff
@@ -1,11 +1,35 @@
 import React from 'react';
 
-const HelloChild = ({ name }) => {
+const HelloChild = ({ title }) => {
     return (
-        <div>
-            <h2>Hello {name}!</h2>
+        <div className="p-4">
+            {/* タイトルの「Form」だけ通常、「テーブル一覧」だけ太字 */}
+            <h1 className="text-xl">
+                <span className="font-normal">Form</span>
+                <span className="font-bold">テーブル一覧</span>
```

Comment:
propsとしてtitleを受け取っているので、それを使用しても良さそうです。
```suggestion
                <span className="font-normal font-bold">{title}</span>
```